lib/kasket/configuration_mixin.rb

```diff
@@ -37,7 +37,8 @@ def kasket_key_for(attribute_value_pairs)
           column = columns_hash[attribute.to_s]
           value = nil if value.blank?
           quoted_value = if column
-            connection.quote(column.type_cast(value), column).downcase
+            casted_value = column.respond_to?(:type_cast_for_database) ? column.cast_type.send(:type_cast, value) : column.type_cast(value)
```

Comment:
maybe an `if column.respond_to?(:type_cast_for_database) # Rails 4.2+` to show why we are doing this